### PR TITLE
added new chain deployments

### PR DESCRIPTION
**Pull request by wtfsayo**

### CODE REVIEW

File: pnpm-lock.yaml
Comment (dan13ram) [inline pnpm-lock.yaml:1]: Is this a mistake? We want this in the repo.

File: packages/ui/src/forms/Select.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ interface SelectProps extends ChakraProps {
   tooltip?: string;
   required?: Required;
   isDisabled?: boolean;
+  _placeholder?: string;
```
Comment (dan13ram) [inline packages/ui/src/forms/Select.tsx:27]: why `_placeholder` while `placeholder` already exists in the default props for `ChakraSelect` ?

File: packages/forms/src/PaymentsForm.tsx [hunk 2]
```diff
@@ -89,11 +104,11 @@ export function PaymentsForm({
   });
 
   useEffect(() => {
-    localSetValue('token', token || _.first(TOKENS)?.address);
-
+    localSetValue('token', nativeWrappedToken);
```
Comment (scottrepreneur) [inline packages/forms/src/PaymentsForm.tsx:107]: this needs to be an object that matches the Select's options, `TOKENS`

File: packages/ui/src/forms/Select.tsx (hunk 1 above)
Comment (wtfsayo) [reply packages/ui/src/forms/Select.tsx:27]: Some typescript thingie

File: pnpm-lock.yaml
Comment (wtfsayo) [reply pnpm-lock.yaml:1]: Have been for a while

File: packages/forms/src/PaymentsForm.tsx (hunk 2 above)
Comment (wtfsayo) [reply packages/forms/src/PaymentsForm.tsx:107]: Umm why? It's just setting the default value of TOKEN changes???

Comment (scottrepreneur) [reply packages/forms/src/PaymentsForm.tsx:107]: Ah you're right

File: packages/ui/src/forms/Select.tsx (hunk 1 above)
Comment (scottrepreneur) [reply packages/ui/src/forms/Select.tsx:27]: change the prop in the component to be `placeholder`. it should match

File: pnpm-lock.yaml
Comment (scottrepreneur) [reply pnpm-lock.yaml:1]: shouldn't need a placeholder with a default value

**Review by scottrepreneur — APPROVED** [review]

**Pull request merged**